src/mongocxx/read_preference.hpp

```diff
@@ -238,6 +238,28 @@ class MONGOCXX_API read_preference {
     ///
     stdx::optional<std::chrono::seconds> max_staleness() const;
 
+    /// Sets the hedge document to be used for the read preference. Sharded clusters running MongoDB
+    /// 4.4 or later can dispatch read operations in parallel, returning the result from the fastest
+    /// host and cancelling the unfinished operations.
+    ///
+    /// This may be an empty document or a document of the form { enabled: <boolean> }.
+    ///
+    /// Hedged reads are automatically enabled in MongoDB 4.4+ when using a ``nearest`` read
+    /// preference. To explicitly enable hedging, the ``hedge`` document must be
```

Comment:
to explicitly... disable?  If it's enabled by default?

Reply:
Changed to "enable or disable".